.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 exclude: ^tests/data/
 repos:
   - repo: https://github.com/PyCQA/flake8
-    rev: 3.8.3
+    rev: 5.0.4
```

Comment:
has been fixed

Reply:
lint will failed if we use flake8 == 3.8.3. It conflicts with `importlib_metadata`
https://github.com/python/importlib_metadata/issues/406